content/challenges/easy/3 - Movies/func.js

```diff
@@ -17,5 +17,5 @@
  * @returns {boolean}
  */
 module.exports = (customerMoney, isMovieFull, rating, age, isParentWith) => {
-    // Your code here
+    return customerMoney >= 8 && !isMovieFull && ((isParentWith || age >= 17) && (rating === "PG" || rating === "R")) || rating === "G";
```

Comment:
So I'm sure this works! Usually a good sanity check to whether or not your return is too complicated is to ask yourself the question "how easy is this to understand?"

Code should actually almost read like well written prose. When your logic is compressed like this it makes it hard for the next developer to edit. If I wanted to add in PG-13 and PG this line would get even longer.

Try pull this back out into if/then/else logic so it reads more like a human would look at it.